go/client/cmd_chat_api.go

```diff
@@ -380,7 +364,7 @@ func (c *CmdChatAPI) sendV1(ctx context.Context, arg sendArgV1) Reply {
 	}
 
 	// find the conversation
-	gilres, err := client.GetInboxLocal(ctx, chat1.GetInboxLocalArg{
+	gilres, err := client.GetInboxAndUnboxLocal(ctx, chat1.GetInboxAndUnboxLocalArg{
```

Comment:
It's using the `conversation.Triple` below, and this is only one TLF. Although I guess it doesn't matter as long as we make sure `TlfName` is from `arg`. I'll change it to use the non-unbox version then